webapp/components/Modal.vue

```diff
@@ -1,39 +1,52 @@
 <template>
   <div class="modal-wrapper">
+    <!-- Todo: Put all modals with 2 buttons and equal properties in one customiced 'danger-action-modal' -->
     <disable-modal
       v-if="open === 'disable'"
       :id="data.resource.id"
       :type="data.type"
       :name="name"
+      :callbacks="data.callbacks"
+      @close="close"
+    />
+    <release-modal
+      v-if="open === 'release'"
+      :id="data.resource.id"
+      :type="data.type"
+      :name="name"
       @close="close"
     />
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-05-27T06:52:30Z" title="Monday, May 27th 2019, 8:52:30 am +02:00">May 27, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -26,6 +33,7 @@
 
 <script>
 import DisableModal from '~/components/Modal/DisableModal'
+import ReleaseModal from '~/components/Modal/ReleaseModal'
```


Path has to be '~/components/ReleaseModal/ReleaseModal‘, since we named the file not `index.js` and moved it in an other folder now!?